components/centraldashboard/public/components/main-page.js

```diff
@@ -566,6 +580,55 @@ export class MainPage extends mixinBehaviors([AppLocalizeBehavior], utilitiesMix
         this.menuLinks = JSON.parse(JSON.stringify(this.menuLinks));
     }
 
+    /**
+     * Handles the list Notebooks Servers response to set date format and icon.
+     * @param {Event} responseEvent
+     */
+    async _onNotebookServersResponse(responseEvent) {
+        const response = responseEvent.detail.response;
+        this.loading = true;
+
+        try {
+            const defaultnotebook = response.notebooks.find((nb)=>
+                nb.labels['notebook.statcan.gc.ca/default-notebook'] &&
+                nb.status.phase === 'stopped'
+            );
+            if (defaultnotebook) {
+                this.startNotebook(defaultnotebook);
+            }
+        } catch (err) {
+            this._onNotebookServersError(err);
+        }
+        this.loading = false;
+    }
+
+    startNotebook(notebook) {
+        // start the notebook
+        fetch(
+            // eslint-disable-next-line max-len
+            `jupyter/api/namespaces/${notebook.namespace}/notebooks/${notebook.name}`, {
+                method: 'PATCH',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify({stopped: false}),
+            }
+        )
+            .then((response)=>{
+                if (!response.ok) {
+                    throw new Error('Failed to start default notebook');
+                }
+            });
```

Comment:
should we have a toast, or a way for the user to see if it fails?

Reply:
Not sure what you mean by toast. If we want to give feedback to users when this fails, we would just need to edit the `_onNotebookServersError(error)` function defined right below. Right now it is just printing to the console, which can help in debugging if this error should come up. But we could also make it display a snackbar with an error message. I wasnt sure what to do since this function is suppose to be an automatic "behind the scenes" kind of process.

Reply:
If we are okay with having a console error, if it happens. Yes snackbar, toast is for android